.bento/config.yml

```diff
@@ -29,20 +29,7 @@ tools:
     - try-except-continue
     - urllib-urlopen
     run: true
-  dlint:
-    ignore: []
-    run: true
   flake8:
     ignore:
     - bare-except-bugbear
     run: true
-  r2c.registry.latest:
```

Comment:
This is running https://r2c.dev/default-r2c-checks, which should be covered by our `semgrep-run-r2c-config` job.